tensorflow/python/ops/math_grad.py

```diff
@@ -767,6 +767,10 @@ def _SparseMatMul(t1, t2, out_dtype,
 def _FloorGrad(_, unused_grad):
   return [None]
 
+@ops.RegisterGradient("Rint")
```

Comment:
What is the final verdict here? just a Comment ? or something like ZeroGradient?